Libraries/src/Amazon.Lambda.DynamoDBEvents/Amazon.Lambda.DynamoDBEvents.csproj

```diff
@@ -3,30 +3,26 @@
   <Import Project="..\..\..\buildtools\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net8.0</TargetFrameworks>
```

Comment:
Removed `netstandard2.0` since it was removed from #1237 as well. Please let me know if I should revert this.